File: AndroidXPatternsApp/jmvp-app/src/main/java/com/woowrale/jmvp/ui/search/local/LocalSearchView.java
package com.woowrale.jmvp.ui.search.local;

import com.woowrale.jmvp.data.model.Contact;
import com.woowrale.jmvp.ui.adapters.ContactsAdapterFilterable;

import java.util.List;

import com.woowrale.jmvp.ui.base.BasePresenter;
import io.reactivex.disposables.CompositeDisposable;

public interface LocalSearchView extends BasePresenter.View {

    CompositeDisposable getDisposable();

    ContactsAdapterFilterable getmAdapter();

    List<Contact> getContactsList();

}

File: KInstantSearch/app/src/main/java/com/woowrale/mvp/ui/search/local/LocalSearchView.kt
package com.woowrale.mvp.ui.search.local

import com.woowrale.mvp.data.model.Contact
import com.woowrale.mvp.ui.base.BasePresenter
import com.woowrale.mvp.ui.adapters.ContactsAdapterFilterable
import io.reactivex.disposables.CompositeDisposable

interface LocalSearchView : BasePresenter.View {

    val disposable: CompositeDisposable

    fun setContactsList(contactList: List<Contact>)

    fun getContactsList():List<Contact>

    fun getmAdapter(): ContactsAdapterFilterable

}